django_celery_extensions/management/commands/run_celery.py

```diff
@@ -37,19 +51,22 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **options):
+        celery_type = options.get('type')
+        celery_settings = options.get('celery_settings')
+        extra_arguments = options.get('extra_args', '')
         if options.get('use_reloader'):
-            self.stdout.write('Starting celery with autoreload...')
+            if os.environ.get(DJANGO_AUTORELOAD_ENV) != 'true':
+                self.stdout.write('Starting celery with autoreload...')
             run_with_reloader(
-                start_celery,
-                celery_type=options.get('type'),
-                celery_settings=options.get('celery_settings'),
-                extra_arguments=options.get('extra_args', ''),
-                autoreload=True
+                autoreload_celery,
+                celery_type=celery_type,
+                celery_settings=celery_settings,
+                extra_arguments=extra_arguments
             )
         else:
             self.stdout.write('Starting celery...')
-            start_celery(
-                celery_type=options.get('type'),
-                celery_settings=options.get('celery_settings'),
-                extra_arguments=options.get('extra_args', '')
+            process = subprocess.Popen(
```

Comment:
if autoreload is turned off we can keep process and call terminate on it if sigterm is called

Reply:
maybe should be possible use this too https://stackoverflow.com/a/65242141 and not run celery as subprocess